docs/getting-started/installation.md

```diff
@@ -0,0 +1,66 @@
+# Installation
+
+## Nix/NixOS
+
+Direct issues installing `chain-bench` via `nix` through the channels mentioned [here](https://nixos.wiki/wiki/Support)
+
+You can use `nix` on Linux or macOS and on other platforms unofficially.
+
+`nix-env --install -A nixpkgs.chain-bench`
+
+Or through your configuration as usual
+
+NixOS:
+
+```nix
+  # your other config ...
+  environment.systemPackages = with pkgs; [
+    # your other packages ...
+    chain-bench
+  ];
+```
+
+home-manager:
+
+```nix
+  # your other config ...
+  home.packages = with pkgs; [
+    # your other packages ...
+    chain-bench
+  ];
+```
+
+## Binary
+
+Download the archive file for your operating system/architecture from [here](https://github.com/aquasecurity/chain-bench/releases/tag/{{ git.tag }}). 
```

Comment:
@06kellyjac the link here is broken because i am not inspecting it with github pages? or is it just typo?

Reply:
it's templated out by mkdocs. I can also try comment this out as a TODO